functions/Sync-DbaAvailabilityGroup.ps1

```diff
@@ -342,7 +346,7 @@ function Sync-DbaAvailabilityGroup {
             if ($Exclude -notcontains "AgentJob") {
                 if ($PSCmdlet.ShouldProcess("Syncing Agent Jobs from $primaryserver to $secondaryservers")) {
                     Write-ProgressHelper -Activity $activity -StepNumber ($stepCounter++) -Message "Syncing Agent Jobs"
-                    Copy-DbaAgentJob -Source $server -Destination $secondaries -Force:$force -Job $Job -ExcludeJob $ExcludeJob
+                    Copy-DbaAgentJob -Source $server -Destination $secondaries -Force:$force -Job $Job -ExcludeJob $ExcludeJob -DisableOnDestination $JobDisableOnDestination
```

Comment:
since this is a switch, it'll need to be `-DisableOnDestination:$DisableJobOnDestination` (gonna fix + switch the word)